test/performance_test/performance.go

```diff
@@ -63,7 +64,16 @@ func main() {
 }
 
 func makeTarget(accessToken string) vegeta.Targeter {
-	url := fmt.Sprintf("%s://%s/api/v1/Patient/$export?_type=%s", proto, apiHost, endpoint)
+	if endpoint != "Patient" {
```

Comment:
I think it might be better to take endpoint as it is passed in. Allows for the test client to be more flexible... i.e., add more endpoints, test error cases (passing in bad endpoints), etc.
(see similar comment below in `bcda_client.go`